demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1369,3 +1372,23 @@ def default_params_have_default_additional_info(self):
                               suggested_fix=Errors.suggest_fix(self.file_path))
             return False
         return True
+
+    def has_no_fromlicense_key_in_contributions_integration(self):
+        pack_name = get_pack_name(self.file_path)
+        if pack_name:
+            metadata_path = Path(PACKS_DIR, pack_name, PACKS_PACK_META_FILE_NAME)
+            metadata_content = self.get_metadata_file_content(metadata_path)
+
+            if metadata_content.get('support', '').lower() == XSOAR_SUPPORT:
+                return True
+
+        conf_params = self.current_file.get('configuration', [])
+        for param_name in conf_params:
+            if 'fromlicense' in param_name.keys():
+                error_message, error_code = Errors.fromlicense_in_parameters(param_name.get('name'))
```

Comment:
Where are you differentiating between xsoar supported and non-xsoar supported if the condition in line 1378 is false?

Reply:
What should be the behavior (if the pack is not in the content repo)
Always false? (because I think there is no way to check the support level).